grunt/usebanner.js

```diff
@@ -13,9 +13,10 @@ module.exports = {
         files: {
             src: [
                 path.join(process.cwd(), 'dist', 'library', 'styles', 'ux-aspects.css'),
+                path.join(process.cwd(), 'dist', 'library', 'styles', 'hpe-icons.min.css'),
```

Comment:
Since we are shipping both, I suppose you should banner the un-minified version as well.